packages/evidence/cli.js

```diff
@@ -185,7 +188,8 @@ prog
 	.action((args) => {
 		populateTemplate();
 		clearQueryCache();
-		buildHelper('VITE_BUILD_STRICT=true npx vite build', args);
```

Comment:
With this change, what happens if I run 
`npm run build:strict` followed by `npm run build`?
would the `VITE_BUILD_STRICT=true` hang around, or would it be cleared because it's a new process?

Reply:
From my testing on `template` I can run `npm run build:strict` then `npm run build` without issue.

Reply:
is the env var cleared?   i.e `npm run build` doesn't trigger an actual strict build?

Reply:
yes, I will send you the demo video.

Reply:
Oh no need, just wanted to be sure we accounted for this in testing - sounds like we did!  I think to get to this part a child subprocess has to triggered so that would make sense.